modules/gmc-agent/src/main/java/org/gridgain/service/config/NodeConfigurationService.java

```diff
@@ -79,20 +67,15 @@ public NodeConfigurationService(GridKernalContext ctx, WebSocketManager mgr) {
     }
 
     /**
-     * @param uuid Uuid.
+     * @param nid Uuid.
      * @param cfgList Config list.
      */
-    boolean onNodeConfiguration(UUID uuid, Object cfgList) {
-        try {
-            IgniteConfigurationWrapper cfg = F.first((List<IgniteConfigurationWrapper>) cfgList);
-            String consistentId = ctx.cluster().get().localNode().consistentId().toString();
-            NodeConfiguration nodeCfg = new NodeConfiguration(consistentId, mapper.writeValueAsString(cfg));
+    boolean onNodeConfiguration(UUID nid, Object cfgList) {
+        String cfg = F.first((List<String>) cfgList);
```

Comment:
Do we need to check for `null` here? `F.first()` can return null.

Reply:
Done.